pkg/controller/plan/context/migration.go

```diff
@@ -147,6 +151,16 @@ func (r *Source) build(ctx *Context) (err error) {
 		return
 	}
 
+	if r.Provider.Type() == v1beta1.OpenShift {
+		client, err := r.Provider.Client(r.Secret)
```

Comment:
maybe
```suggestion
		if r.Provider.IsHost() {
		    client, err := r.Provider.Client(nil)
		 }
```

Reply:
or not, based on:
https://github.com/kubev2v/forklift/blob/main/pkg/apis/forklift/v1beta1/provider.go#L120-L124